module_2/assignment1_1.js

```diff
@@ -0,0 +1,8 @@
+const PI = 3.147;
```

Comment:
Add comment what happen if we access const  and let before assignment

Reply:
Done